source/elements/oneDAL/source/algorithms/clustering/kmeans_init.rst

```diff
@@ -0,0 +1,62 @@
+.. highlight:: cpp
+.. default-domain:: cpp
+
+======================
+K-Means initialization
+======================
+
+The K-Means initialization algorithm gets :math:`n` feature vectors as input and
+chooses :math:`k` initial centroids. After initialization, K-Means algorithm
```

Comment:
Don't you want to mention how it chooses centroids? Randomly or not?

Reply:
Depends on method, so. no